bluetooth/bluetooth-control

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+# Set up variables
+VOLUME_UP_GPIO=$(</usr/src/volume_up_gpio)
+
+BLUETOOTH_GPIO=$(</usr/src/bluetooth_gpio)
+
+VOLUME_DOWN_GPIO=$(</usr/src/volume_down_gpio)
+
+NEXT_GPIO=$(</usr/src/next_gpio)
+
+PREVIOUS_GPIO=$(</usr/src/previous_gpio)
+
+OUTPUT_GPIO=$(</usr/src/output_gpio)
+
+TIMER="0"
+
+SKIP_STOP="0"
+
+BLUETOOTH_DISCOVERABLE="0"
+
+# Set up + volume
+if [[ ! -d /sys/class/gpio/gpio$VOLUME_UP_GPIO ]]; then
+  echo "$VOLUME_UP_GPIO" > /sys/class/gpio/export
+  sleep 1 
+fi
+echo "in" > /sys/class/gpio/gpio$VOLUME_UP_GPIO/direction
+
+# Set up bluetooth
+if [[ ! -d /sys/class/gpio/gpio$BLUETOOTH_GPIO ]]; then
+  echo "$BLUETOOTH_GPIO" > /sys/class/gpio/export
+  sleep 1 
+fi
+echo "in" > /sys/class/gpio/gpio$BLUETOOTH_GPIO/direction
+
+# Set up - volume
+if [[ ! -d /sys/class/gpio/gpio$VOLUME_DOWN_GPIO ]]; then
+  echo "$VOLUME_DOWN_GPIO" > /sys/class/gpio/export
+  sleep 1 
+fi
+echo "in" > /sys/class/gpio/gpio$VOLUME_DOWN_GPIO/direction
+
+# Set up next
+if [[ ! -d /sys/class/gpio/gpio$NEXT_GPIO ]]; then
+  echo "$NEXT_GPIO" > /sys/class/gpio/export
+  sleep 1 
+fi
+echo "in" > /sys/class/gpio/gpio$NEXT_GPIO/direction
+
+# Set up previous
+if [[ ! -d /sys/class/gpio/gpio$PREVIOUS_GPIO ]]; then
+  echo "$PREVIOUS_GPIO" > /sys/class/gpio/export
+  sleep 1 
+fi
+echo "in" > /sys/class/gpio/gpio$PREVIOUS_GPIO/direction
+
+# Set up utput gpio (there is one energy input missing)
+if [[ ! -d /sys/class/gpio/gpio$OUTPUT_GPIO ]]; then
+  echo "$OUTPUT_GPIO" > /sys/class/gpio/export
+  sleep 1 
+fi
+echo "out" > /sys/class/gpio/gpio$OUTPUT_GPIO/direction
```

Comment:
We could eliminate a lot of code using a function here. Haven't tested it, but this should be ok:

```suggestion
function configure_gpio() {
  local GPIO_PIN="$1"
  local DIRECTION="$2"

  if [[ ! -d /sys/class/gpio/gpio$GPIO_PIN ]]; then
    echo "$GPIO_PIN" > /sys/class/gpio/export
    sleep 1 
  fi
  
  echo "$DIRECTION" > /sys/class/gpio/gpio$GPIO_PIN/direction
}

configure_gpio "$VOLUME_UP_GPIO" "in"
configure_gpio "$BLUETOOTH_GPIO" "in"
configure_gpio "$VOLUME_DOWN_GPIO" "in"
configure_gpio "$NEXT_GPIO" "in"
configure_gpio "$PREVIOUS_GPIO" "in"
configure_gpio "$OUTPUT_GPIO" "out"
```